### ISSUE TITLE
dynamic pagelist

### ISSUE THREAD

**Issue by mosely1** [dokufreaks/plugin-pagelist#129]
It would be great if pagelist could dynamically list all pages in a namespace without having to explicitly list them. I realize this is what the plugin dir is meant to do, but dir seems bloated and unmaintained. Pagelist is great, I just need it to dynamically include and exclude pages as those pages get created or deleted in the namespace. Thanks!

**Comment by plastes-de** [comment]
Yes, good idea, and additionally a page break after <n> items will be helpful. Thanks!

**Comment by Klap-in** [comment]
Please one request per issue report :-) (please explain in that new issue what you mean with the page break)

**Comment by ajshea** [comment]
indexmenu plugin does this

**Comment by mosely1** [comment]
indexmenu only gives an index of pages. Our wiki uses indexmenu for the sidebar as a navigation tool. 

What my users additionally want is a master-control page, located at each :start: page, that not only lists the pages in the namespace but also gives the description, title, and date of last edit for each page in the namespace. indexmenu cannot do this. We currently accomplish this goal with pagelist, but it is tedious when pages get created, deleted, or moved. 

Thank you again for consideration.

**Comment by ajshea** [comment]
That actually would be a very useful tool, I can think of places where I would like to use something like that.

**Comment by Klap-in** [comment]
The dir plugin uses pagelist, in my understanding to do this. What about the plugins https://www.dokuwiki.org/plugin:nslist or https://www.dokuwiki.org/plugin:simplenavi?

**Comment by mosely1** [comment]
Simplenavi is great for a sidebar. I use it for that. And love it for that. 

Nslist is similar to simplenavi in that it only provides the hierarchy of pages and namespaces (and optionally the date) but not the description of the pages. 

I need my Master Control index pages to include a description (abstract) of each page in the namespace, and Pagelist is excellent for that.

**Comment by mosely1** [comment]
YES! https://github.com/dokufreaks/plugin-pagelist/issues/106 That person was trying to do exactly what I would like to do.
